### ISSUE TITLE
[BITV] 9.3.3.2/2.3 - Header / menus "Search" and "Search contacts" - Search input fields lack visual labels and there are no search buttons or symbols to help identify them as search fields. (9)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37085]
see also https://github.com/nextcloud/server/issues/37091

- [x] https://github.com/nextcloud/server/pull/33826
- [x] https://github.com/nextcloud/server/pull/40314

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v2-n3

</details>

- [x] 9\.3.3.2/3.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v3-n1
- [x] 9\.3.3.2/4.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v4-n1
- [x] 9\.3.3.2/5.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v5-n1
- [x] 9\.3.3.2/6.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v6-n1
- [x] 9\.3.3.2/7.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v7-n1
- [x] 9\.3.3.2/8.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v8-n1
- [x] 9\.3.3.2/9.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v9-n1
- [x] 9\.3.3.2/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v10-n1

**Comment by JuliaKirschenheuter** [comment]
input fields can be covered by https://github.com/nextcloud-libraries/nextcloud-vue/pull/4394. All places have to be adjusted to new component. "submit" button with "magnifying glass" can be helpful to identify them as search fields

**Comment by jancborchardt** [comment]
@JuliaKirschenheuter yep, magnifying glass on the left sounds good.

**Comment by Pytal** [comment]
> @JuliaKirschenheuter yep, magnifying glass on the left sounds good.

Not implementing due to increase of legacy tech debt

**Comment by Pytal** [comment]
Can you confirm that the visual labels
![image](https://github.com/nextcloud/server/assets/24800714/d4ecf538-4b79-46d2-aa80-471152da87d5)
![image](https://github.com/nextcloud/server/assets/24800714/cc9a702c-fc51-4997-9c45-67406a462083)
are sufficient @michaelnissenbaum ? Or are icons absolutely required?

**Comment by michaelnissenbaum** [comment]
I can confirm that the solution using <label> elements meets the accessibility criteria. Looks good to me.
